authorization-dotnet/BouncyCastleCryptography/AsymmetricEncryption/DsaEncryptor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Org.BouncyCastle.Crypto.Generators;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Security;
+using System.Text;
+
+namespace BouncyCastleCryptography.AsymmetricEncryption
+{
+    public static class DsaEncryptor
+    {
+        public static AsymmetricCipherKeyPair GenerateDsaKeyPair()
+        {
+            // Create DSA parameters
+            DsaParametersGenerator dsaParamsGenerator = new DsaParametersGenerator();
```

Comment:
For variable declaration we use var, so the above line must be rewritten as follows:
var dsaParamsGenerator = new DsaParametersGenerator();

This comment is application throughout the project